### DISCUSSION TITLE
Ability to fork a repo on mobile app.

### DISCUSSION THREAD

**Discussion by europa1613** [community/community#27232] [Actions]
Hello, am I missing something? Is there an ability to fork a repo on mobile app?

**Comment by johnmichaelarc** [comment]
**hope this helps**
# Forking a Repo on Mobile
1. Find a repo that you like
2. Open your browser menu
3. Turn on the Desktop site
4. Fork that Repo
5. Done!

**Reply by europa1613** [reply]
Hello, I am on iOS GitHub App. And what do you mean by browser menu?

**Reply by johnmichaelarc** [reply]
I'm on iOS too, what i mean is forking from the actual app is not yet possible. But when i'm trying to fork something, i'm copy/pasting the link to safari to fork it. that's my way.

**Reply by europa1613** [reply]
Got you! Thanks for the reply !!!

**Reply by nexuspcs** [reply]
> I'm on iOS too, what i mean is forking from the actual app is not yet possible. But when i'm trying to fork something, i'm copy/pasting the link to safari to fork it. that's my way.

Same method for me too. 
GitHub needs to introduce a native way into the app though....

**Comment by jfullstackdev** [comment]
yup, you should be using a standard browser to do so
